docs/examples/grafana/README.md

```diff
@@ -0,0 +1,237 @@
+# Grafana - All-in-one
+
+In this example, we are going to use [Prometheus](https://prometheus.io/) and [Tempo](https://grafana.com/oss/tempo/), both displayed inside [Grafana](https://grafana.com/grafana/).
+Just like [Jaeger example](../jaeger-docker/README.md), we will need to setup a collector to gather both metrics and traces.
+
+### Project setup
+
+Configure the project using your favorite tool, once again setup is similar to the Jaeger example, only difference being that we export metrics as they will be stored inside Prometheus:
+
+@:select(build-tool)
+
+@:choice(sbt Scala 2)
+
+Add settings to the `build.sbt`:
+```scala
+ThisBuild / scalaVersion := "3.3.0"
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "otel4s-java" % "@VERSION@", // <1>
+  "io.opentelemetry" % "opentelemetry-exporter-otlp" % "@OPEN_TELEMETRY_VERSION@" % Runtime, // <2>
+  "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % "@OPEN_TELEMETRY_VERSION@" % Runtime // <3>
+)
+run / fork := true
+javaOptions += "-Dotel.java.global-autoconfigure.enabled=true" // <4>
+javaOptions += "-Dotel.service.name=grafana-example"           // <5>
+```
+
+@:choice(sbt Scala 3)
+
+Add settings to the `build.sbt`:
+```scala
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "otel4s-java" % "@VERSION@", // <1>
+  "io.opentelemetry" % "opentelemetry-exporter-otlp" % "@OPEN_TELEMETRY_VERSION@" % Runtime, // <2>
+  "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % "@OPEN_TELEMETRY_VERSION@" % Runtime // <3>
+)
+run / fork := true
+javaOptions += "-Dotel.java.global-autoconfigure.enabled=true" // <4>
+javaOptions += "-Dotel.service.name=grafana-example"           // <5>
+```
+
+@:choice(scala-cli Scala 2)
+
+Add directives to the `grafana.scala`:
+```scala
+//> using lib "org.typelevel::otel4s-java:@VERSION@" // <1>
+//> using lib "io.opentelemetry:opentelemetry-exporter-otlp:@OPEN_TELEMETRY_VERSION@" // <2>
+//> using lib "io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:@OPEN_TELEMETRY_VERSION@" // <3>
+//> using `java-opt` "-Dotel.java.global-autoconfigure.enabled=true" // <4>
+//> using `java-opt` "-Dotel.service.name=grafana-example"           // <5>
+```
+
+@:choice(scala-cli Scala 3)
+
+Add directives to the `grafana.scala`:
+```scala
+//> using lib "org.typelevel::otel4s-java:@VERSION@" // <1>
```

Comment:
```suggestion
//> using scala 3.3.0
//> using lib "org.typelevel::otel4s-java:@VERSION@" // <1>
```

That way we can ensure users will have a correct Scala version and the example will compile